day-4/ques-13.js

```diff
@@ -0,0 +1,23 @@
+//1Problem Statement: Write a program to return inverse of an array
+
+const arr = [3, 4, 2, 0, 1];
+const inverseArr = new Array(arr.length);
+
+const makeInverseArray = (arr) => {
+  for (i in arr) {
+    inverseArr[arr[i]] = i;
+  }
+};
+
+const checkInverse = (arr, inverseArr) => {
```

Comment:
what is the use of this function

Reply:
The question is about how can we inverse an array:
Inverse of an array means that if in an array we swap the value of the array at a particular index with the index of the element, and then we obtain the same array then we can call it as the irreversible array.
eg: [3,4,2,0,1]
in this: when we swap the array elements with the index values in another array:
invArr = [,,,,]
then referring the original array, we place 0 value at index 3 in invArray
invArr = [,,,0,]
Similarly when we refer the index 1 of original array we place 1 at index 4 i.e.
invArr = [,,,0,1],
similarly referring the index 2,3,4 we get the inverse array: invArr = [3,4,2,0,1]